src/js/providers/video-actions-mixin.ts

```diff
@@ -62,7 +62,7 @@ const VideoActionsMixin: VideoActionsInt = {
         // object-fit is not implemented in IE or Android Browser in 4.4 and lower
         // http://caniuse.com/#feat=object-fit
         // feature detection may work for IE but not for browsers where object-fit works for images only
-        const fitVideoUsingTransforms = Browser.ie || (OS.iOS && OS.version.major < 9) || Browser.androidNative;
+        const fitVideoUsingTransforms = Browser.ie || (OS.iOS && OS.version.major && OS.version.major < 9) || Browser.androidNative;
```

Comment:
```suggestion
        const fitVideoUsingTransforms = Browser.ie || (OS.iOS && (OS.version.major | 0) < 9) || Browser.androidNative;
```

You could use an OR operator but the bitwise op is a shortcut for floor (converts anything to an int).

Reply:
<img width="106" alt="Screen Shot 2021-04-08 at 2 40 51 PM" src="https://user-images.githubusercontent.com/333258/114079658-761c5d80-9878-11eb-9e2e-303ab191cf60.png">